packages/datagateway-download/src/index.tsx

```diff
@@ -93,6 +93,13 @@ export const fetchSettings = (): Promise<DownloadSettings | void> => {
         );
       }
 
+      // Ensure all fileCountMax and totalSizeMax are present.
+      if (!('fileCountMax' in settings && 'totalSizeMax' in settings)) {
+        throw new Error(
+          'fileCountMax or totalSizeMax is undefined in settings'
+        );
+      }
+
```

Comment:
Following changes made in https://github.com/ral-facilities/datagateway/pull/1398, this needs to be removed